frontend/src/data/variables/AcsPopulationProvider.test.ts

```diff
@@ -187,6 +192,56 @@ describe("AcsPopulationProvider", () => {
     );
   });
 
+  test("State and Race Breakdown with standard race filter", async () => {
+    const rawData = [
+      stateRow(AL, RACE, TOTAL, 2),
+      stateRow(AL, RACE, ASIAN, 2),
+      stateRow(NC, RACE, TOTAL, 20),
+      stateRow(NC, RACE, ASIAN, 5),
+      stateRow(NC, RACE, WHITE, 15),
+      // Non-standard, will be excluded from the non-standard filter
+      stateRow(NC, RACE, "White", 17),
+      stateRow(NC, RACE, NON_HISPANIC, 13),
+    ];
+
+    const datasetId = "acs_population-by_race_state_std";
+    dataFetcher.setFakeDatasetLoaded(datasetId, rawData);
+
+    let breakdowns = Breakdowns.forFips(new Fips(NC.code)).andRace();
+    let response = await new AcsPopulationProvider().getData(
+      new MetricQuery("population", breakdowns)
+    );
+    expect(response).toEqual(
+      new MetricQueryResponse(
+        [
+          finalRow(NC, RACE, TOTAL, 20, 100),
+          finalRow(NC, RACE, ASIAN, 5, 25),
+          finalRow(NC, RACE, WHITE, 15, 75),
+          finalRow(NC, RACE, "White", 17, 85),
+          finalRow(NC, RACE, NON_HISPANIC, 13, 65),
+        ],
+        [datasetId]
+      )
+    );
+
+    breakdowns = Breakdowns.forFips(new Fips(NC.code)).andRace(
+      onlyIncludeStandardRaces()
+    );
+    response = await new AcsPopulationProvider().getData(
+      new MetricQuery("population", breakdowns)
+    );
+    expect(response).toEqual(
```

Comment:
I wonder if we should update createWithAndWithoutTotalEvaluator to return a function that takes an Array of BreakdownFilters and forEach them instead of hard coding the total/not total dichotomy

Reply:
Yeah I thought about that, but I worried about over-generalizing a test utility that becomes hard to maintain. It's easy to have those things get out of control (I say this from personal failures lol). I think it can be valuable if done carefully but requires some thought with the best way to design